python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -574,18 +577,53 @@ def current_in_flight_tasks(self) -> int:
             for actor_state in self._running_actors.values()
         )
 
-    def scale_up(self, num_actors: int) -> int:
+    def can_scale_down(self):
```

Comment:
why not putting the debouncing logic in the autoscaler?
It's also part of the autoscaling policy

Reply:
Went back and forth on it decided to keep it in AP, since it owns the logic of actually scaling up and tracking most recent actions